draft-linker-digital-emblem.md

```diff
@@ -212,6 +230,23 @@ However, marking entire networks should drastically ease the burden of deploymen
 Notably, a digital emblem cannot be applied to infrastructure that is used for both services worthy and unworthy of protection.
 A digital emblem must always identify assets that only serve purposes that enjoy protection under IHL.
 
+# Proposed Designs for a Digital Emblem {#proposals}
+
+Two designs were proposed to solve the problem of a digital emblem.
+First, JHU APL proposed a DNS-based solution for a digital emblem {{?I-D.haberman-digital-emblem}}.
+Second, CECYT (a joint endeavour between ETH Zurich and Bonn University) proposed *An Authentic Digital EMblem (ADEM)*, which was initially described in an academic publication {{ADEM}}, but was also specified technically {{ADEM-SPEC}} and has openly accessible, working prototypes ([Go library and CLI](https://github.com/adem-wg/adem-proto), [Browser extension](https://github.com/adem-wg/adem-chrome)).
+We focus here on ADEM as it was selected by the ICRC as the most suitable proposal for a digital emblem.
+
+ADEM was designed following the requirements laid out in this draft.
+ADEM follows a decentralized trust model and utilizes existing infrastructure wherever possible, e.g., it provides accountability through the Certificate Transparency infrastructure {{?RFC6962}}.
+ADEM was designed to be distributed over many channels and as such is not bound to one mode of transportation.
+The academic paper foresees distribution of digital emblems via TLS, DNS, and UDP.
+Next to its prototype, ADEM's security was thoroughly evaluated and formal proofs of security are described in the academic publication.
+
+Once there is consensus on the scope of a digital emblem, the plan is to propose ADEM as a basis for designing a digital emblem.
+Although ADEM was evaluated on many occasions, it is subject to change.
```

Comment:
```suggestion
Although ADEM has gone through several iterations in collaboration with the ICRC, it is expected that it will be refined by the IETF WG that adopts it to ensure industry interoperability with good protocol practices.
```
I'm not insisting on this literal text change word for word, but I hope it illustrates the point I'm suggesting (being direct about what I know to be true, which is "this is no rough draft, but we're also open to change as part of the multistakeholder standards development process"

Reply:
Reads much better than my version and is exactly what I could not but wanted to express! :)